ldap-api/src/main/java/ca/bc/gov/hlth/ldapapi/service/LdapService.java

```diff
@@ -83,6 +87,36 @@ private boolean checkUserLocked(Attributes attributes) {
         return unlocked;
     }
 
+    private boolean checkUserPasswordExpired(Attributes attributes) {
+        Attribute passwordChangeDate = attributes.get(LDAP_ATTR_PASSWORD_CHANGE_DATE);
+        Attribute passwordLifespan = attributes.get(LDAP_ATTR_PASSLIFESPAN);
+        boolean expired = false;
+
+        if (passwordChangeDate != null) {
+            try {
+                Long changedDaysAgo = Long.valueOf(passwordChangeDate.get().toString());
```

Comment:
This isn't really "changedDaysAgo" is it? Changed days ago would be a number like "32", as in "the password was changed 32 days ago". How about `dateLastChangedOn` or something like that?